src/components/App.js

```diff
@@ -37,18 +38,20 @@ class App extends React.Component {
 
   render() {
     return (
-      <div className="ui container">
-        <SearchBar onFormSubmit={this.onTermSubmit} />
-        <div className="ui grid">
-          <div className="ui row">
-            <div className="eleven wide column">
-              <VideoDetail video={this.state.selectedVideo} />
-            </div>
-            <div className="five wide column">
-              <VideoList
-                onVideoSelect={this.onVideoSelect}
-                videos={this.state.videos}
-              />
+      <div className="app">
+        <div className="  ui container">
```

Comment:
Nit: remove white space

Reply:
Also, are `ui` and `container` also semantic UI classes; not sure what ones you were getting rid of?

Reply:
The whole app component is styled with Semantic UI.